### DISCUSSION TITLE
Naming files does not work

### DISCUSSION THREAD

**Discussion by Gevaarlijk** [aclap-dev/video-downloadhelper#90] [Help and Q&A]
### Your question

Naming files does not work properly:

Setting:

https://i.vgy.me/Tyv3iJ.png

Download:

https://i.vgy.me/ckqQwi.png

Using 8.2.0.23

**Comment by paulrouget** [comment]
Noted. I'll look into it.

**Comment by 98gmarquee** [comment]
I'm noticing the same. Some download just a a string of numbers, the rest have the words all sepearted with hyphones along with the 480p/240p/360p as the last word in the file name. In the past, I'd only see that on a few random sites, but now I'm seeing it on all of them.

**Reply by paulrouget** [reply]
Can you help me reproduce the issue?

- do you use smartnaming?
- from which site do you download?
- which action do you use? Quick Download? Download?

**Reply by paulrouget** [reply]
@98gmarquee this should be fixed in version 8.2.1.0. Upcoming soon.

**Comment by Gevaarlijk** [comment]
Does someone know if and how I can install an older version that doesn't have this problem? It's unworkable now.

**Reply by paulrouget** [reply]
Help me understand what exactly the problem is:

- do you use smartnaming?
- from which site do you download?
- which action do you use? Quick Download? Download?

**Reply by Gevaarlijk** [reply]
No smartnaming. I've tested that in the past. It did no seem to do anything and there is no explanation on how it should work.

Site: https://www.nporadio2.nl/uitzendingen

I don't know which action I use. I have to click an episode. Start it, pause it then I can click on the extension Icon right of the taskbar and then click on the filename and then it downloads. This is the only way it works.

**Reply by paulrouget** [reply]
ok - that specific website, I fixed it in 8.2.0.24. Hope to publish that new version reaaaally soon.

**Comment by GrampaWildWilly** [comment]
This may be closing the barn door after the cows have left but . . .

I went to the NPO2 web site & it looked like this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/3e29afa0-9af9-408b-abeb-6dd9d9d4b807)

This is nothing to be alarmed about.  Some sites are like this.  This is why it says on the VDH menu to click play.  So I clicked play.

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/2a6378bc-f383-4971-8dca-30dbd88606df)

That does not bode well.  But OK.  I enabled DRM & clicked play again.  It played for maybe a second, then went silent.  The button that had been a play button remained a stop button, as if it thought it was playing something.  But I wasn't hearing anything.  However, VDH did recognize something:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6023b55c-095f-4721-a913-9466caf1e797)

OK.  So I launched a download on the indicated variant.  This is what the blue dot VDH status menu was telling me:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/abc68eb4-2f2e-4eda-9706-a05227f463bf)


That time indicator kept getting larger.  I suppose that is no surprise since this is a livestream.  I let it run for a few minutes & stopped it in that status menu by clicking on the X in the black dot, as indicated in that image.

What I got was an mp3 of duration only 31 seconds.  It played fine, but I had only 31 seconds of it.  I have no idea if the DRM protection was causing me trouble.  Also, I suppose we should revisit this after we get the new version of VDH that Paul just mentioned.

**Reply by GrampaWildWilly** [reply]
I just tried this again after installing the new 8.2.0.25a1 beta for Firefox.  This time the radio station came on & played until I stopped it.  That's better than before when it played only about 1 second & stopped, although it looked like it was still playing.  But the download with VDH now gets a Side Download error after just a few seconds.  I tried several times, with Reload extension after each failure.  No joy.

**Reply by paulrouget** [reply]
ah? I'm surprised… it should have worked.

**Reply by paulrouget** [reply]
Dang… it's failing again. Sorry… thought I fixed it. Looking again…

**Comment by Gevaarlijk** [comment]
It works fine when downloading past broadcasts instead of the livestream though. Is VDH supposed to be able to download live streams at all?

**Reply by paulrouget** [reply]
It should, yes, but it will not show any progress as it's infinite size.

**Comment by GrampaWildWilly** [comment]
Just retried with VDH 8.2.1.0a1 beta.  No change.

**Reply by paulrouget** [reply]
Are you saying the naming issue is not fixed in the beta?

**Reply by GrampaWildWilly** [reply]
No, I'm getting the same failure to launch the download.  It hesitates a moment while it thinks about it.  Then I get this:

```
SideDownload error:

moz-extension://9d469d39-7727-4f85-8beb-0761511221b9/background/main.js:1
ms@moz-extension://9d469d39-7727-4f85-8beb-0761511221b9/background/main.js:1:37415
Tn@moz-extension://9d469d39-7727-4f85-8beb-0761511221b9/background/main.js:1:37473
gs@moz-extension://9d469d39-7727-4f85-8beb-0761511221b9/background/main.js:1:37543
lT@moz-extension://9d469d39-7727-4f85-8beb-0761511221b9/background/main.js:44:515
```

The name it proposes before it fails is this:

NPO Radio 2.mp3

The URL I'm using is this: https://www.nporadio2.nl/
